applications/DemStructuresCouplingApplication/custom_python/dem_structures_coupling_python_application.cpp

```diff
@@ -36,8 +36,10 @@ PYBIND11_MODULE(KratosDemStructuresCouplingApplication, m)
     AddCustomUtilitiesToPython(m);
 
     //registering variables in python
-    
+
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, DEM_SURFACE_LOAD)
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, DEM_SURFACE_LOAD_OLD)
+    KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, DEM_SURFACE_LOAD_OLD2)
```

Comment:
Is there a standard nomenclature for successive 'OLD's? I used OLD_OLD in the past. We should think about it

Reply:
I hope this implementation will be temporary, because we should be able to damp out the structure instead of smoothing the external loads on it. I would vote it for leaving the current name and remove these variables as soon as we manage to damp out the structure by means of good parameters in the bossak scheme of the structure...